index.html

```diff
@@ -1,14 +1,192 @@
-<!DOCTYPE HTML>
+<!DOCTYPE html>
 <html>
-  <head>
-    <title>Mockup Home</title>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
-  </head>
-  <body>
-    <section id="header"></section>
-    <section id="features"></section>
-    <section id="main"></section>
-    <section id="footer"></section>
-  </body>
+
+<head>
+  <title>Mockup Home</title>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
+  <link rel="stylesheet" href="styles/style.css" />
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Bangers&display=swap" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap" rel="stylesheet">
+</head>
+
+<body>
+  <section id="header">
+    <h1 class="header" id="header-main">HOMEPAGE</h1>
+    <h3 class="header" id="header-sub">What a great homepage</h3>
+    <div class="h-break">
+  </section>
+  <section id="features">
+    <!-- header and three images with captions, with "tell me more button" -->
+    <p class="header" id="feature-header">TAKE A LOOK AT THIS AMAZING <u>HOME PAGE</u>!</p>
+    <div id="image-container">
+      <figure>
+        <img class="feature-image" />
+        <figcaption>image one</figcaption>
+      </figure>
+      <figure>
+        <img class="feature-image" />
+        <figcaption>image two</figcaption>
+      </figure>
+      <figure>
+        <img class="feature-image" />
+        <figcaption>image three</figcaption>
+      </figure>
+    </div>

```

Comment:
The images contained in your `image-container` are pushing out of the size of your container which is causing issues with your "tell me more" button living inside of your quote block

Reply:
Also adjusting the size of the images would help maintain the white space on the right and left of the page